src/__tests__/homepage.test.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import '@testing-library/jest-dom';
+import {cleanup, render, screen} from '@testing-library/react';
+import HomePage from "../pages/Home/Home.page";
+import youtubeVideos from "../mocks/youtube-videos";
+
+describe('Home page Tests', () => {
+  afterEach(cleanup);
```

Comment:
No need to do this, as [that's done automatically](https://testing-library.com/docs/react-testing-library/api/#cleanup) when using `Jest`.